arkane/encorr/data.py

```diff
@@ -28,391 +28,24 @@
-        'N-H': 0.06, 'N-N': -0.23, 'N=N': -0.37, 'N#N': -0.64
-    },
-
-    'cbs-qb3': {
-        'C-H': -0.11, 'C-C': -0.30, 'C=C': -0.08, 'C#C': -0.64, 'O-H': 0.02, 'C-O': 0.33, 'C=O': 0.55,
-        # Table IX: Petersson GA (1998) J. of Chemical Physics, DOI: 10.1063/1.477794
-        'N-H': -0.42, 'C-N': -0.13, 'C#N': -0.89, 'C-F': 0.55, 'C-Cl': 1.29, 'S-H': 0.0, 'C-S': 0.43,
-        'O=S': -0.78,
-        'N=O': 1.11, 'N-N': -1.87, 'N=N': -1.58, 'N-O': 0.35,
-        # Table 2: Ashcraft R (2007) J. Phys. Chem. B; DOI: 10.1021/jp073539t
-        'N#N': -2.0, 'O=O': -0.2, 'H-H': 1.1,  # Unknown source
-    },
-
-    'cbs-qb3-paraskevas': {
-        # NOTE: The Paraskevas corrections are inaccurate for non-oxygenated hydrocarbons,
-        # and may do poorly in combination with the Petersson corrections
-        'C-C': -0.495, 'C-H': -0.045, 'C=C': -0.825, 'C-O': 0.378, 'C=O': 0.743, 'O-H': -0.423,
-        # Table2: Paraskevas, PD (2013). Chemistry-A European J., DOI: 10.1002/chem.201301381
-        'C#C': -0.64, 'C#N': -0.89, 'C-S': 0.43, 'O=S': -0.78, 'S-H': 0.0, 'C-N': -0.13, 'C-Cl': 1.29,
-        'C-F': 0.55,  # Table IX: Petersson GA (1998) J. of Chemical Physics, DOI: 10.1063/1.477794
-        'N-H': -0.42, 'N=O': 1.11, 'N-N': -1.87, 'N=N': -1.58, 'N-O': 0.35,
-        # Table 2: Ashcraft R (2007) J. Phys. Chem. B; DOI: 10.1021/jp073539t
-        'N#N': -2.0, 'O=O': -0.2, 'H-H': 1.1,  # Unknown source
-    },
-
-    # Identical corrections for 'b3lyp/cbsb7', 'b3lyp/6-311g(2d,d,p)', 'b3lyp/6-311+g(3df,2p)', 'b3lyp/6-31g(d,p)'
-    'b3lyp/cbsb7': {
-        'C-H': 0.25, 'C-C': -1.89, 'C=C': -0.40, 'C#C': -1.50,
-        'O-H': -1.09, 'C-O': -1.18, 'C=O': -0.01, 'N-H': 1.36, 'C-N': -0.44,
-        'C#N': 0.22, 'C-S': -2.35, 'O=S': -5.19, 'S-H': -0.52,
-    },
-    'b3lyp/6-311g(2d,d,p)': {
-        'C-H': 0.25, 'C-C': -1.89, 'C=C': -0.40, 'C#C': -1.50,
-        'O-H': -1.09, 'C-O': -1.18, 'C=O': -0.01, 'N-H': 1.36, 'C-N': -0.44,
-        'C#N': 0.22, 'C-S': -2.35, 'O=S': -5.19, 'S-H': -0.52,
-    },
-    'b3lyp/6-311+g(3df,2p)': {
-        'C-H': 0.25, 'C-C': -1.89, 'C=C': -0.40, 'C#C': -1.50,
-        'O-H': -1.09, 'C-O': -1.18, 'C=O': -0.01, 'N-H': 1.36, 'C-N': -0.44,
-        'C#N': 0.22, 'C-S': -2.35, 'O=S': -5.19, 'S-H': -0.52,
-    },
-    'b3lyp/6-31g(d,p)': {
-        'C-H': 0.25, 'C-C': -1.89, 'C=C': -0.40, 'C#C': -1.50,
-        'O-H': -1.09, 'C-O': -1.18, 'C=O': -0.01, 'N-H': 1.36, 'C-N': -0.44,
-        'C#N': 0.22, 'C-S': -2.35, 'O=S': -5.19, 'S-H': -0.52,
-    },
-
-}
-
-# Melius-type bond additivity correction parameters
-mbac = {}
+import importlib.util
+import os
+
+from rmgpy import settings
+
+quantum_corrections_path = os.path.join(settings['database.directory'], 'quantum_corrections', 'data.py')
+spec = importlib.util.spec_from_file_location("quantum_calculations", quantum_corrections_path)
+data = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(data)
```

Comment:
I like this approach because it's a neat way of loading data and having it immediately available. Do you think there are any issues with storing the data at the module level? I guess the other option would be writing some kind of database class that would store the data as attributes. Would that be better?

Reply:
I thought about this as well, but in theory no one should be modifying corrections data at runtime, so I think storing at the module level should be fine. I am fine implementing a quick database structure, but I am not sure that it is needed. Worthwhile discussion though

Reply:
Generally agree, although I guess the `write_to_database` code in my BAC PR does modify at runtime, if desired, although it uses the same mechanism as you use here to reload the data.